### ISSUE TITLE
[BUG] Media element rotation on Android does not work as expected

### ISSUE THREAD

**Issue by cagriy** [CommunityToolkit/Maui#1610]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Did you read the "Reporting a bug" section on Contributing file?

- [X] I have read the "Reporting a bug" section on Contributing file: https://github.com/CommunityToolkit/Maui/blob/main/CONTRIBUTING.md#reporting-a-bug

### Current Behavior

When a MediaElement is rotated, on iOS both the video frame and video content are rotated together, e.g. a 90-degree rotation gives a landscape orientation without changing the app's orientation.

On Android, however, although the video frame is rotated correctly, the content stays as is.

iOS: 
<img width="375" alt="image" src="https://github.com/CommunityToolkit/Maui/assets/1134069/340bd775-b99b-432e-b159-7f0477358305">

Android:
<img width="542" alt="image" src="https://github.com/CommunityToolkit/Maui/assets/1134069/49ffe983-0666-4abc-94f3-7e7cddc5b34d">

### Expected Behavior

It is expected that on Android both video frame and the content rotate together, similar to iOS implementation.

### Steps To Reproduce

1. Download repo
2. Build and run on iOS simulator
3. Run on Android simulator

### Link to public reproduction project repository

https://github.com/cagriy/MediaElementBug

### Environment

```markdown
- .NET MAUI CommunityToolkit: 3.0.1
- OS: MacOs 14.0
- .NET MAUI: 7 and 8
```

**Comment by ne0rrmatrix** [comment]
@vhugogarcia I can verify this as a bug on android. It is as described and needs to be fixed.

**Comment by jfversluis** [comment]
Can we verify this is still a thing on the Media3 implementation of this?

**Comment by ne0rrmatrix** [comment]
@jfversluis Yes I can confirm it is still an issue. Unfortunately it is a limittation of the default configuration for the video render. It is not recommended to change it but it can be done. It would affect performance and has been discussed in the past in monthly standup. I don't remember what we decided atm. The current renderer that is default is `SurfaceView` The one we would need to add for rotating video's or transparencies to work is `TextureView` which is available as an option. If anything we can discuss it again and go over the pro's and cons of adding it or not.
